static/templates/actors/parts/actor/actor-info.html

```diff
@@ -2,32 +2,43 @@
 <div><span class="pusher"></span>
   <span class="item-title">{{localize "TWODSIX.Actor.Items.TRAITS"}}</span>
   <div class="items-traits gear-header">
-    <span class="item-name">{{localize "TWODSIX.Actor.Items.Name"}}</span>
-    <span class="item-name">{{localize "TWODSIX.Actor.Skills.Level"}}</span>
-    <span class="item-name centre">{{localize "TWODSIX.Actor.Items.ShortDescr"}}</span>
```

Comment:
Do we really want to get rid of the short description?  I think there are some traits that don't translate directly into effects.

Reply:
Good question. I felt as though it got really crowded having both description and all effects there. What would be your preference? Would a "if no effects show description" work?

Reply:
That sounds like a good balance. Show effect if it exists otherwise the short description.  I'll take another pass through tomorrow. Thanks

Reply:
Ended up adding back the short description. I think it actually works quite okay with both of them. Thanks for the suggestion!